deploy/charts/istio-csr/templates/certificate.yaml

```diff
@@ -5,9 +5,18 @@ metadata:
   namespace: istio-system
 spec:
   dnsNames:
+  {{- if .Values.app.istio.revisions }}
+  {{- $revisions := dict -}}
```

Comment:
Is there any need for this dict here?

Reply:
Thanks @keithmattix, looks like I ended up having some lines go missing. Should be better now with proper duplicate checking.

Reply:
Excellent - thanks for adding that fix. LGTM!